src/oz/access/ownable.cairo

```diff
@@ -0,0 +1,115 @@
+    }
+
+    #[embeddable_as(OwnableImpl)]
+    impl Ownable<
+        TContractState, +HasComponent<TContractState>
+    > of interface::IOwnable<ComponentState<TContractState>> {
+        /// Returns the address of the current owner.
+        fn owner(self: @ComponentState<TContractState>) -> ContractAddress {
+            self.Ownable_owner.read()
+        }
+
+        /// Transfers ownership of the contract to a new address.
+        fn transfer_ownership(
+            ref self: ComponentState<TContractState>, new_owner: ContractAddress
+        ) {
+            assert(!new_owner.is_zero(), Errors::ZERO_ADDRESS_OWNER);
+            self.assert_only_owner();
+            self._transfer_ownership(new_owner);
+        }
+
+        /// Leaves the contract without owner. It will not be possible to call `assert_only_owner`
+        /// functions anymore. Can only be called by the current owner.
+        fn renounce_ownership(ref self: ComponentState<TContractState>) {
+            self.assert_only_owner();
+            self._transfer_ownership(Zeroable::zero());
+        }
+    }
+
+    /// Adds camelCase support for `IOwnable`.
+    #[embeddable_as(OwnableCamelOnlyImpl)]
+    impl OwnableCamelOnly<
+        TContractState, +HasComponent<TContractState>
+    > of interface::IOwnableCamelOnly<ComponentState<TContractState>> {
+        fn transferOwnership(ref self: ComponentState<TContractState>, newOwner: ContractAddress) {
+            self.transfer_ownership(newOwner);
+        }
+
+        fn renounceOwnership(ref self: ComponentState<TContractState>) {
+            self.renounce_ownership();
+        }
+    }
+
+    #[generate_trait]
+    impl InternalImpl<
+        TContractState, +HasComponent<TContractState>
+    > of InternalTrait<TContractState> {
+        /// Sets the contract's initial owner.
+        ///
+        /// This function should be called at construction time.
+        fn initializer(ref self: ComponentState<TContractState>, owner: ContractAddress) {
+            self._transfer_ownership(owner);
+        }
+
+        /// Panics if called by any account other than the owner. Use this
+        /// to restrict access to certain functions to the owner.
+        fn assert_only_owner(self: @ComponentState<TContractState>) {
+            let owner: ContractAddress = self.Ownable_owner.read();
+            let caller: ContractAddress = get_caller_address();
+            assert(!caller.is_zero(), Errors::ZERO_ADDRESS_CALLER);
+            assert(caller == owner, Errors::NOT_OWNER);
+        }
+
+        /// Transfers ownership of the contract to a new address.
+        ///
+        /// Internal function without access restriction.
+        fn _transfer_ownership(
+            ref self: ComponentState<TContractState>, new_owner: ContractAddress
+        ) {
+            let previous_owner: ContractAddress = self.Ownable_owner.read();
+            self.Ownable_owner.write(new_owner);
```

Comment:
Shouldn't `assert_only_owner();` be called somewhere in here?

Reply:
this is OpenZeppelin code. I completely removed it from the repo, it is now only a dependency.